text/0032-nft-standard.md

```diff
@@ -117,7 +191,26 @@ The AttributeObject may have more attributes than the ones described above but t
             "type": "ring",
             "value": 0
         }
-    ]
+    ],
+    "royalty": {
+        "fee_basis_points": 500,
+        "creators": [
+            {
+              "address": "H7t2eNhFNeH3hrUAJj4AAu9amtWmPT4fLB",
+              "share": 9500
+            },
+            {
+              "address": "HNBuYEsJYznh5zZGx4eDn2NXYuiPpgKBgH",
+              "share": 500
+            },
+            {
+              "address": "RaNd0MsJYznh5zYOyoDn2NXYuiPAdDre5s",
+              "share": 0
```

Comment:
Explain below that this address is identified as a creator of the NFT, even though it will not receive any share of royalties.

Reply:
Done